### ISSUE TITLE
Go 1.10.x breaks version prompt layout

### ISSUE THREAD

**Issue by spotlightishere** [caiogondim/bullet-train.zsh#278]
The introduction of Go 1.10.x gave a second digit to the version number.

https://github.com/caiogondim/bullet-train.zsh/blob/06f1d86ae2a407cf24a833ee892e6f9c748283ff/bullet-train.zsh-theme#L539

only expects one digit as shown below, causing the prompt to newline:
```
> go version | grep --colour=never -oE '[[:digit:]].[[:digit:]]'
1.1
0.2
```

**Comment by salmanulfarzy** [comment]
Duplicate of #255, Fixed with #264

**Comment by spotlightishere** [comment]
Ah, thanks. I did a terrible search beforehand it seems.
